src/pyscript/cli.py

```diff
@@ -55,6 +55,8 @@ def version() -> None:
 )
 _show_option = typer.Option(None, help="Open output file in web browser.")
 
+_title_option = typer.Option(None, help="Add title to html file")
```

Comment:
We should be able to put the default value here instead of handling it in the subcommand function.